### DISCUSSION TITLE
Missing target-attribute in events at V5 - new method for disable controls at active multiselection?

### DISCUSSION THREAD

**Discussion by RunRanger** [fabricjs/fabric.js#7960] [Q&A] [answered]
Hello together,

i'm currently trying to disable scaling for fabric objects. This works for single objects:
```
object.setControlsVisibility({
      tr: false,
      tl: false,
      br: false,
      bl: false,
      ml: false,
      mt: false,
      mr: false,
      mb: false,
    })
```

But with multiselection the whole setting of the selection is default again (which makes of course sense)
I've seen many approaches with events like 
```
canvasToBind.on('selection:created', (event) => {
    const target = event.target;
   //disable Controls with target-attribute
}
```

But it seems, that target isn't an attribute of event anymore since V5. Is there any other way to access the selection-controls?

**Comment by RunRanger** [comment, marked as answer]
After several trys i finally got it.

You have to use the `selected` attribute of the selection:created event. There you take the first element and go to its group-attribute:
```
canvasToBind.on('selection:created', (event: any) => {
        const { selected } = event;
        if (selected.length > 1)
          setGroupConfigs(selected[0].group)
}
```

**Reply by ShaMan123** [reply]
In future releases I hope to make ActiveSelection manageable. It wasn't because Group couldn't support it but now it can.
Look at #7882
